src/components/campaign/campaign-body.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { FC } from "react";
+import { CampaignMatchPOST200Data } from "../../core/dpl-cms/model";
+
+export interface CampaignBodyProps {
+  campaignData: CampaignMatchPOST200Data;
+}
+
+const CampaignBody: FC<CampaignBodyProps> = ({ campaignData }) => {
+  return (
+    <section className="campaign">
+      {campaignData.image && campaignData.image.url && (
+        <img
+          className={`campaign__image ${
+            !campaignData.text ? "campaign__image--full-width" : ""
+          }`}
+          src={campaignData.image.url}
+          alt={campaignData.image.alt}
+        />
+      )}
+      <h4 className="campaign__title campaign__title--ellipsis">
+        {campaignData.text}
+      </h4>
```

Comment:
Please make the title conditional of the `text` property.

It seems like this will add an empty title element even if is an image-only campaign. That does not seem right.

Reply:
Of course! How did I not see this myself.. 
Thanks 😉